perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AuthzResolverBlImpl.java

```diff
@@ -80,6 +80,7 @@
 import cz.metacentrum.perun.core.impl.Utils;
 import cz.metacentrum.perun.core.implApi.AuthzResolverImplApi;
 import cz.metacentrum.perun.registrar.model.Application;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
NITPICK: We usually want to import the smallest part of a package and then use these methods directly without the prefix `StringUtils.`.
```suggestion
import org.apache.commons.lang3.StringUtils.isBlank;
```

Reply:
Should be fixed